test/0.6/PasskeyAccount.t.sol

```diff
@@ -5,13 +5,15 @@ pragma solidity ^0.8.13;
 import { Test } from "forge-std/Test.sol";
```

Comment:
Test structure should follow source code. It should be `test/account/0.6/PasskeyAccount`